packages/graphql/src/index.ts

```diff
@@ -17,5 +17,8 @@
  * limitations under the License.
  */
 
+import { upperFirst } from "graphql-compose";
```

Comment:
IMO it's not GraphQL's composes responsibility and kinda confusing. But I get why we have done this to reduce dependencies.

Reply:
I get it! But definitely no point in having the `upper-case-first` dependency, and after our own `upperCaseFirst` function had a bug in it, I would rather use one from a "trusted source".

In `utils/index.ts` we could do:

```ts
import { upperFirst } from "graphql-compose";
export { upperFirst }
```

Because then you could do `import { upperFirst } from "./utils"` which feels a _bit_ nicer to use, if you like?